src/SIL.LCModel/DomainImpl/OverridesCellar.cs

```diff
@@ -3994,56 +3986,21 @@ private ITsString GetFeatureValueString(bool fLongForm)
 			return tisb.GetString();
 		}
 
-		internal ITsString GetFeatureValueStringSorted()
+		private string FeatureSpecKey(IFsFeatureSpecification spec)
 		{
-			var tisb = TsStringUtils.MakeIncStrBldr();
-			var iCount = FeatureSpecsOC.Count;
-			if (iCount > 0)
+			if (spec.FeatureRA == null)
+				return "";
```

Comment:
I'm not sure when exactly this would get hit, but it will mean this spec will be sorted to the front, and if there's more than one spec with a null RA then they will not be consistently ordered. I'd either have a fallback ordering or return something like `"2"+spec.Guid` to ensure that they're always ordered consistently.